ios/FluentUI.Demo/FluentUI.Demo/Demos/ColorDemoController.swift

```diff
@@ -153,6 +153,11 @@ class ColorDemoController: UIViewController {
             if let navigationController = navigationController {
                 navigationController.popViewController(animated: false)
                 colorThemeHost?.updateToWindowWith(type: windowType, pushing: self)
+                UIAccessibility.post(notification: .screenChanged, argument: self.segmentedControl.segmentView(at:segmentedControl.selectedSegmentIndex))
+                DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(100))
```

Comment:
Why use a timer?

Reply:
From [here](https://stackoverflow.com/questions/21554056/call-ios-accessibility-voice-over-programmatically-on-a-ui-element): "If you post your announcement immediately after a user action, such as a button tap, it may get swallowed by the button's automatic voice-over announcement. In this case you should post your announcement after a suitable delay (as mentioned here). If that delay is too short, your announcement might cut into the button's announcement."

We'll be taking an alternative route to fix this bug though, so the timer won't be used in future iterations of this change.